Assignment4/JuanSantacruzz-21A286/task06.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1XWhGqVSRdtbNXKJOUnT3VCxCE3AC-N_y
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This will not run in Python. You need to comment out line 12. The same thing applies for other tasks